pkg/grafana/grafana.go

```diff
@@ -102,6 +128,122 @@ func DeleteByID(ctx context.Context, grafanaAPI *client.GrafanaHTTPAPI, id int64
 	return nil
 }
 
+func ConfigureDefaultDatasources(ctx context.Context, grafanaAPI *client.GrafanaHTTPAPI, organization Organization) ([]Datasource, error) {
+	logger := log.FromContext(ctx)
+
+	// TODO using a serviceaccount later would be better as they are scoped to an organization
```

Comment:
I'm.actually not sure it would bé best as we might have a lot of service accounts and so on